sdk/src/main/java/io/opentelemetry/sdk/trace/TraceProtoUtils.java

```diff
@@ -1,180 +0,0 @@
-/*
- * Copyright 2019, OpenTelemetry Authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package io.opentelemetry.sdk.trace;
-
-import com.google.common.annotations.VisibleForTesting;
-import com.google.protobuf.ByteString;
-import com.google.protobuf.Timestamp;
-import io.opentelemetry.proto.trace.v1.Span;
-import io.opentelemetry.proto.trace.v1.Span.Attributes;
-import io.opentelemetry.proto.trace.v1.Span.Links;
-import io.opentelemetry.proto.trace.v1.Span.SpanKind;
-import io.opentelemetry.proto.trace.v1.Span.TimedEvents;
-import io.opentelemetry.sdk.internal.TimestampConverter;
-import io.opentelemetry.sdk.resources.Resource;
-import io.opentelemetry.trace.AttributeValue;
-import io.opentelemetry.trace.Link;
-import io.opentelemetry.trace.Span.Kind;
-import io.opentelemetry.trace.SpanContext;
-import io.opentelemetry.trace.SpanId;
-import io.opentelemetry.trace.Status;
-import io.opentelemetry.trace.TraceId;
-import io.opentelemetry.trace.Tracestate;
-import java.util.Collection;
-import java.util.Map;
-
-// Utilities to convert Span SDK to proto representation of the Span.
-final class TraceProtoUtils {
```

Comment:
Move this class for the moment into:
"exporters/otproto" - this is where we will have the official opentelemetry protocol implemented.

Maybe make it public so that utils like toProtoTraceId can be reused by Jaeger.

Reply:
sure thing. I'll move it over there, if it'll be handy for the OT protocol.

Reply:
done